foundation_cms/templates/patterns/components/button.html

```diff
@@ -0,0 +1,12 @@
+{% if button_style == "btn-primary" or button_style == "btn-secondary" %}
```

Comment:
Same HTML structure as before, but with class name updates.

File has also been renamed from `link_button.html` to `button.html`